docs/core_docs/vercel.json

```diff
@@ -61,6 +61,18 @@
       "source": "/docs/integrations/llms/togetherai(/?)",
       "destination": "/docs/integrations/llms/together/"
     },
+    {
+      "source": "/docs/tutorials/query_analysis(/?)",
+      "destination": "/docs/tutorials/rag#query-analysis/"
```

Comment:
I don't think this hash will do anything since this happens on the backend - not sure though

Reply:
Appears to correctly re-direct in preview: https://langchainjs-docs-git-cc-updatetutorials-langchain.vercel.app/docs/tutorials/query_analysis